### ISSUE TITLE
MobileNetV2 model performance

### ISSUE THREAD

**Issue by d-li14** [KaiyangZhou/deep-person-reid#106]
@KaiyangZhou Could you please share the detailed training arguments to achieve your previously published MobileNetV2 performance (e.g. 84.2 (65.8) on Market1501 in https://github.com/KaiyangZhou/deep-person-reid/blob/bc13a9fdc10f2e5f1240bb7aa8db1fc74e32caa7/MODEL_ZOO.md)?
BTW, why do you remove it from model_zoo afterwards? This result is not stable?

**Comment by KaiyangZhou** [comment]
Try this https://github.com/KaiyangZhou/deep-person-reid/issues/7

You might wanna use the imagenet-pretrained model, which can give you better performance.

**Comment by d-li14** [comment]
Thanks very much. I have noticed #7 and combined the following suggestions
1. set a larger learning rate e.g. 0.01
2. train longer for 300 epochs and decay the lr once at the 100 epoch
3. train from scratch and use label smooth technique
4. keep the other arguments in the script as default

As a result the best result I got is still rank-1 ~75%, mAP ~55% on Market1501.

I will try to load the pretrained backbone to improve the performance soon. But I am curious about the `MODEL_ZOO.md` you released before, how did you get that high performance in the history?
